### PR TITLE
Repost Figure blog article

### PR DESCRIPTION
**Pull request by hflynn**
As discussed on https://github.com/openmicroscopy/www.openmicroscopy.org/pull/194 this migrates http://figure.openmicroscopy.org/2015/04/30/quality-figures-from-omero-figure.html to the main OME blog. Dated for release tomorrow, I don't think it needs much re-checking.

N.B. For checking with a local build today, you'll need to add "future: true" to your local config

### CODE REVIEW

**Comment by hflynn** [comment]
@will-moore

**Comment by will-moore** [comment]
I haven't managed to build this locally yet - need to install some more dependencies and upgrade ruby etc.. but the text looks great.

**Comment by hflynn** [comment]
Have you tried just deleting your local Gemfile if you have one? That fixed some issues for me

**Comment by will-moore** [comment]
Unfortunately didn't help.
Upgraded ruby with ```brew install ruby```
Then
```
$ sudo gem install jekyll-seo-tag
...
26 gems installed
$ jekyll serve
Configuration file: /Users/wmoore/Desktop/BLOG/blog/_config.yml
  Dependency Error: Yikes! It looks like you don't have jekyll-seo-tag or one of its dependencies installed. In order to use Jekyll as currently configured, you'll need to install this gem. The full error message from Ruby is: 'cannot load such file -- jekyll-seo-tag' If you run into trouble, you can find helpful resources at http://jekyllrb.com/help/! 
jekyll 3.1.2 | Error:  jekyll-seo-tag
```

**Comment by hflynn** [comment]
Ah, not an expert on getting this stuff working I'm afraid! It builds fine for me though so if you're happy to sign off on the text, I'll just merge and release it. Otherwise you're welcome to come and look at my local build (I'd take a screenshot but I'd need about 4 to get it all in)

**Comment by will-moore** [comment]
The text looks great, so I'm happy for you to go ahead.

File: _posts/2018-02-08-figure.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title:  Quality figures from OMERO.figure
+categories: tools
+---
+
+*This is a repost from the Figure.openmicroscopy.org blog where its creator
```
Comment (jburel) [inline _posts/2018-02-08-figure.md]: figure.o.o

File: _posts/2018-02-08-figure.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+intensity range and channels can be toggled on and off. None of these
+changes will alter the original microscope data.
+OMERO.figure does not require you to save 8-bit images before
+creating a figure, since all rendering is done 'live' within
+the figure app itself after importing images, as described below.
+
+
+## Figure layout
+
+OMERO.figure is similar to Inkscape and Adobe Illustrator in that it
```
Comment (jburel) [inline _posts/2018-02-08-figure.md]: need for copyright or trademark symbol?

File: _posts/2018-02-08-figure.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+## Exporting final figure files
+
+OMERO.figure offers export in PDF and TIFF formats. Both are generated on the
+OMERO server using a Python script and the Python Imaging Library (PIL) for
+image manipulation. Figures are saved on the server and are then available to download.
+
+
+### Creating TIFF Images
+
+Tiff images, at 300 dpi are generated by resampling all the embeded images
```
Comment (jburel) [inline _posts/2018-02-08-figure.md]: embedded

File: _posts/2018-02-08-figure.md (hunk 2 above)
Comment (hflynn) [reply _posts/2018-02-08-figure.md]: We didn't have it originally

Comment (hflynn) [reply _posts/2018-02-08-figure.md]: Looked it up and there's no requirement to do so and it has no legal significance anyway

Comment (jburel) [reply _posts/2018-02-08-figure.md]: thanks

**Pull request merged**